src/Pods/Data/Map_Field_Values.php

```diff
@@ -703,6 +704,85 @@ public function image_fields( $field, $traverse, $field_data, $obj ) {
 		return pods_traverse( $traverse_params, $value );
 	}
 
+	/**
+	 * Map the date field value.
+	 *
+	 * @since 2.9.14
+	 *
+	 * @param string                  $field      The first field name in the path.
+	 * @param string[]                $traverse   The list of fields in the path excluding the first field name.
+	 * @param null|Field|Object_Field $field_data The field data or null if not a field.
+	 * @param Pods|null               $obj        The Pods object or null if not set.
+	 *
+	 * @return null|mixed The date field value or null if there was no match.
+	 */
+	public function date_fields( $field, $traverse, $field_data, $obj = null ) {
+
+		// Default date field handlers.
+		$date_fields = array(
+			'date',
+			'time',
+			'datetime',
+		);
+
+		// Skip if the field exists.
+		if ( $field_data && ! in_array( $field_data->get_type(), $date_fields, true ) ) {
```

Comment:
If we are dealing with post_date etc, those fields will match an object field and be skipped.